### PR TITLE
Card 1 icon component

### PR DESCRIPTION
**Pull request by mistyb01**
## Description

This PR contains `CardOne.tsx`, a component for cards of the "Card 1 Icon" type.
- CardOne can be customized to include a button at the bottom, so it can be used for both "Card 1 with Icon" and "Card 1 with Icon + Button."
	- Add/remove the `buttonText` prop as needed! May need an extra prop for the button's link.

 Plus, two components for cards in general:
- `CardRowContainer.tsx `lays out cards in a flexbox for responsiveness.
- `IconContainer.tsx` takes an icon as a prop, and puts it inside a green circle.

## Related Issue

[Card One](https://app.asana.com/0/0/1204566664333016/f)

## Acceptance Criteria

- [x] Installed Material Icons
- [x] Create a reusable Card, IconContainer component
- [x] Make it responsive on different screen sizes
- [x] Apply global styles defined in `theme.ts`

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓ | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
| ✓ | :link: Update dependencies |
|     | :scroll: Docs              |

## Testing Steps / QA Criteria

- git pull
- git checkout card-1-icon-component
- npm i
- to test out the card components:
- in volunteer.jsx, add these imports:

```
import CardRowContainer from '../components/cards/CardRowContainer';
import CardOne from '../components/cards/CardOne'
import AccessibilityIcon from '@mui/icons-material/Accessibility';
```

- within the return statement include the following code:
```
<CardRowContainer>
  <CardOne
    title="Make a Donation"
    description="A tax-deductible donation below helps us provide our Partners with the tools they need."
    icon={<AccessibilityIcon fontSize="large" sx={{ color: "white" }} />}
    buttonText="Donate"
    buttonLink="#"
  />
  <CardOne
    title="Sponsor Us"
    description="Help our Partners increase their nonprofit’s impact by becoming an ally and offering support."
    icon={<AccessibilityIcon fontSize="large" sx={{ color: "white" }} />}
    buttonText="Sponsor"
    buttonLink="#"
  />
  <CardOne
    title="Share Our Mission"
    description="Spread the word with your friends, coworkers, and social media, to help continue our mission."
    icon={<AccessibilityIcon fontSize="large" sx={{ color: "white" }} />}
    buttonText="Share"
    buttonLink="#"
  />
</CardRowContainer>
```

### CODE REVIEW

File: components/cards/CardOne.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    title?: string,
+    description: string,
+    icon: ReactNode,
+    buttonText?: string,
+    buttonLink?: string
+}
+
+const CardOne = ({title, description, icon, buttonText, buttonLink}: CardOneProps) => {
+    return (
+        <Card sx={{
```
Comment (wheninseattle) [inline components/cards/CardOne.tsx:20]: I didn't know about this way of using breakpoints with MUI. Thanks for showing me!

File: pages/partner.jsx [hunk 2]
```diff
@@ -8,6 +8,10 @@ import TextFormGroup from '../components/form/TextFormGroup'
 import TextFormGroupWithAddon from '../components/form/TextFormGroupWithAddon'
 import Layout from '../components/Layout'
 
+import CardRowContainer from '../components/cards/CardRowContainer';
+import CardOne from '../components/cards/CardOne'
+import AccessibilityIcon from '@mui/icons-material/Accessibility';
```
Comment (wheninseattle) [inline pages/partner.jsx:13]: It looks like these were accidentally left here.

File: pages/partner.jsx [hunk 3]
```diff
@@ -39,6 +43,33 @@ export default function PartnerForm() {
...
+
+<CardOne
+  title="Share Our Mission"
+  description="Spread the word with your friends, coworkers, and social media, to help continue our mission."
+  icon={<AccessibilityIcon fontSize="large" sx={{ color: "white" }} />}
+  buttonText="Share"
+  buttonLink="#"
+/>
+
+</CardRowContainer>
```
Comment (wheninseattle) [inline pages/partner.jsx:72]: It looks like these were accidentally left here.

**Review by wheninseattle — COMMENTED** [review]
This looks great, Misty! I like your break down of components and your use of the theme.

File: components/cards/CardOne.tsx (hunk 1 above)
Comment (mistyb01) [reply components/cards/CardOne.tsx:20]: Yeah I found out in the MUI docs! We can also modify the breakpoints in the theme file if needed 👍

File: pages/partner.jsx (hunk 3 above)
Comment (mistyb01) [reply pages/partner.jsx:72]: Yep I was just testing out stuff-- will remove both chunks of code!

**Review by wheninseattle — COMMENTED** [review]
Thanks for making those changes!

**Review by santina — APPROVED** [review]
Just one non-blocking comment. I made come changes to the repo in #15 
- Could you run `yarn` to update the yarn.lock file, and remove `package-lock.json`. I moved the use of npm to use yarn
- Could you run save (command + s) on all the tsx files. It'd reformat the files if you're using vscode.  If not you can run `prettier --write components/cards` from the root of the folder. Feel free to change the tab size to 4 in `.prettierrc.json` and `settings.json` if you prefer.

awesome work I love how break down the card components and detailed description of the PR 💯

**Pull request merged**